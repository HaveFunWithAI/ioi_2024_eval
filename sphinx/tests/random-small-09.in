50 876
18 8 32 9 0 1 34 11 10 3 20 29 38 13 36 6 23 46 39 15 41 44 17 27 7 33 37 40 42 19 26 49 2 12 47 21 24 48 45 30 5 14 16 43 22 31 28 25 4 35
32 44
2 32
20 44
28 44
20 25
28 47
4 32
9 28
15 47
1 2
28 46
14 15
20 30
14 26
2 36
4 17
15 24
13 46
22 47
18 32
20 40
10 30
3 32
10 41
20 31
0 2
2 19
12 17
21 30
43 47
17 35
24 34
24 45
14 29
8 43
12 33
13 48
18 42
5 44
22 27
14 38
9 16
37 38
11 40
7 26
4 39
2 6
47 49
23 39
13 31
28 34
31 41
5 46
2 26
0 46
22 34
8 27
13 26
7 20
13 47
13 18
11 38
28 39
1 7
2 39
6 23
20 36
12 37
25 28
3 14
23 46
24 38
11 35
12 48
17 29
22 33
15 42
17 19
11 27
2 45
9 48
29 43
36 40
0 29
26 28
2 23
19 45
14 43
12 41
0 47
33 44
32 43
8 34
19 44
26 29
4 33
21 40
11 25
15 30
3 9
5 24
1 3
17 44
18 38
6 16
33 43
23 32
11 30
7 23
10 43
36 42
7 42
15 43
12 49
27 44
43 45
1 9
34 36
33 35
0 21
28 49
23 48
2 17
15 35
5 40
3 37
25 45
2 42
9 27
23 47
25 32
12 32
9 30
10 20
10 17
6 42
29 39
2 48
1 35
7 45
35 38
33 37
11 39
18 28
26 31
9 39
11 48
10 31
2 25
21 43
29 31
12 38
29 42
29 49
22 40
18 40
14 48
16 23
1 37
12 46
30 34
16 20
8 26
3 41
31 46
34 44
38 45
8 44
18 29
15 41
10 44
1 45
3 44
24 42
18 37
17 40
9 49
33 49
3 22
16 18
37 43
0 49
5 16
5 48
34 49
3 46
31 36
5 27
29 37
1 38
4 45
10 32
30 31
0 26
5 28
16 26
18 47
17 43
24 35
1 43
7 49
22 29
8 32
6 47
18 48
15 49
30 42
27 37
20 38
0 22
10 34
15 18
1 46
18 45
10 23
21 31
16 21
23 27
13 38
35 43
40 46
6 12
25 36
7 36
0 9
17 27
13 25
11 47
8 40
5 13
4 38
22 46
26 36
31 44
17 26
13 17
13 16
36 45
24 46
4 22
7 10
12 36
40 41
3 38
13 22
4 15
15 25
6 17
15 16
9 17
2 20
1 26
0 34
29 32
34 41
9 14
4 48
1 11
2 12
29 38
20 33
1 10
39 46
42 44
2 21
15 22
11 24
1 42
27 35
15 46
19 32
21 38
23 25
8 12
14 19
45 46
6 39
5 19
11 42
22 45
17 41
35 37
16 34
25 49
31 47
12 13
5 7
21 44
19 41
18 27
17 22
10 33
10 18
4 49
7 8
31 40
39 49
19 48
3 33
6 28
1 22
0 16
42 43
1 8
18 23
7 28
1 33
21 29
8 9
20 34
19 29
9 25
4 12
0 44
2 10
40 49
38 40
9 20
41 42
20 22
4 40
1 18
16 17
15 39
23 44
2 7
33 36
11 33
17 24
10 36
13 41
9 43
30 40
28 30
10 49
9 46
4 8
11 34
33 38
25 29
3 35
15 26
7 46
14 34
8 35
7 43
23 33
16 39
16 44
2 15
30 33
16 41
18 26
4 20
18 19
26 43
22 41
24 27
30 35
32 47
24 28
28 38
7 19
21 27
4 42
13 20
28 41
1 27
43 44
41 49
6 11
8 33
30 41
5 17
37 49
26 34
20 46
2 28
41 47
9 26
17 21
5 8
33 40
8 41
19 33
10 45
3 30
30 46
36 41
35 47
0 43
25 41
21 37
41 45
14 30
19 42
9 44
26 33
17 36
25 31
7 11
23 24
3 7
32 45
11 26
8 23
39 48
31 33
12 40
9 38
7 47
18 31
9 32
24 36
17 20
4 26
6 41
12 24
5 41
15 34
1 24
35 36
8 11
16 29
7 14
0 32
15 31
7 30
31 37
21 39
3 11
2 47
15 48
13 29
3 24
27 29
11 32
41 48
21 46
2 35
20 41
1 32
25 30
4 10
6 36
13 40
6 32
32 35
4 43
30 47
10 26
39 41
3 16
38 43
19 27
22 28
16 27
24 33
27 38
6 33
32 40
15 44
13 32
7 37
0 48
26 45
3 25
12 42
0 38
14 47
15 20
16 33
40 47
5 21
16 36
19 22
6 38
9 13
3 48
22 26
16 45
5 23
23 37
7 13
1 5
20 43
42 45
8 30
23 30
16 40
17 38
28 48
19 35
1 36
5 33
35 48
5 25
4 28
2 4
34 42
10 46
4 31
10 42
1 49
5 12
11 21
10 22
9 23
17 31
38 41
4 21
14 25
0 8
15 38
8 39
8 45
3 40
15 40
4 25
25 26
25 43
21 24
21 48
29 46
20 49
16 32
13 36
6 19
15 27
12 26
18 49
23 29
24 37
10 11
20 29
27 47
25 40
15 19
7 40
8 46
4 36
33 45
11 19
37 47
29 44
1 39
9 40
22 38
17 39
24 30
2 41
3 45
13 30
16 47
18 44
8 36
14 18
8 38
12 25
33 41
5 20
2 30
10 15
46 47
46 49
2 14
20 32
31 39
2 24
41 44
11 36
17 48
40 48
21 47
29 30
23 34
27 28
16 46
16 48
7 17
18 43
0 35
29 35
44 48
31 34
8 22
6 31
26 35
18 39
8 18
28 35
18 24
29 36
2 18
37 40
3 20
27 39
10 25
18 46
8 42
28 45
39 47
32 42
11 41
18 35
14 27
15 28
0 36
45 48
35 39
26 38
11 49
11 28
22 35
12 31
6 30
21 34
42 49
7 16
39 42
5 9
5 32
24 39
26 44
29 41
34 39
26 27
3 5
41 46
7 32
3 4
17 23
16 43
15 23
23 38
13 33
22 39
0 30
15 37
6 15
46 48
0 18
6 34
27 40
25 27
26 42
32 46
22 23
6 8
18 33
22 49
9 29
7 39
27 30
11 23
19 23
13 28
2 22
24 43
30 48
6 10
25 34
9 47
43 46
3 10
1 19
16 22
38 48
14 46
13 21
9 12
20 48
0 41
12 47
31 43
0 14
31 42
26 37
37 39
25 48
19 49
0 28
5 37
28 40
28 37
9 37
21 49
6 21
0 7
10 37
3 31
23 41
28 32
28 31
18 25
30 36
6 45
45 49
5 22
21 22
2 13
2 37
6 35
10 13
4 5
26 48
43 49
11 13
12 30
1 25
34 48
16 31
13 35
1 20
2 40
2 33
10 38
5 11
25 42
12 22
17 42
33 34
42 46
1 29
19 24
2 38
25 35
17 45
18 41
29 48
19 20
7 18
14 41
1 15
0 17
12 16
0 37
11 12
25 46
12 35
23 26
8 13
9 33
34 40
26 47
1 16
30 39
27 42
13 24
1 28
32 48
21 32
44 47
11 46
5 43
32 34
21 28
20 27
23 43
25 47
37 44
21 42
14 17
37 46
3 36
14 20
31 45
4 11
8 21
1 23
15 45
4 6
7 38
8 29
13 19
11 43
14 36
13 15
18 22
13 49
1 17
11 18
24 29
13 39
33 46
21 25
12 34
3 6
34 43
6 26
14 16
3 29
39 43
32 49
39 40
6 25
29 47
5 14
10 12
23 31
30 49
36 43
19 30
1 34
9 34
27 32
14 37
28 29
9 18
5 34
6 9
17 37
20 42
12 15
2 46
6 22
2 44
11 22
35 46
4 46
2 9
10 14
14 23
3 18
38 44
3 8
19 46
36 38
36 49
6 14
0 5
6 46
16 28
40 43
9 21
14 45
9 22
22 42
3 39
14 44
6 27
15 17
16 25
27 41
7 21
19 47
1 41
20 23
4 35
7 24
21 23
0 12
6 37
4 37
8 48
19 34
0 45
14 49
27 36
12 45
0 31
27 45
11 44
35 41
30 44
28 33
4 34
9 45
2 29
42 48
12 18
40 44
7 35
2 11
34 46
1 6
