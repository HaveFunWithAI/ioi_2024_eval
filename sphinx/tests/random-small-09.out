50 416
18 8 32 9 0 1 34 11 10 3 20 29 38 13 36 6 23 46 39 15 41 44 17 27 7 33 37 40 42 19 26 49 2 12 47 21 24 48 45 30 5 14 16 43 22 31 28 25 4 35
